### PR TITLE
Adding Symbol Map File

### PR DESCRIPTION
**Pull request by jpc-ae**
SVG sprites are unnecessary, since the use tag works on any kind of file. The only advantage a sprite file has is that it is viewable directly. This can be added to a map file if needed.

### CODE REVIEW

**Comment by jpc-ae** [comment]
I signed it!

**Comment by shyndman** [comment]
Hey @jpc-ae,

Looks awesome! Thanks for bringing this technique to our attention.

I'm going to script this up for repeatability. In the meantime, could you add a note to the root README.md similar to what we have for sprite sheets?

File: iconmaps/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Our recommended way to use symbol sheets is to create them with the subset of icons needed for the project. A good option for creating your own symbol map is to use [gulp-svgstore](https://github.com/w0rm/gulp-svgstore) with individual development icons. This will also allow you to include any other svgs in the project, like a logo, in a single reference file.
+
+## Using provided symbol maps
+
+Material design icons come with an SVG symbol map for each icon category we include. These can be found in the `iconmaps` directory. The full set of current icons is in `MaterialIcons.svg`.
+
+To add an icon, place the following in your document wherever it is needed:
+
+```HTML
+<svg class="svg-ic_24px">
```
Comment (shyndman) [inline iconmaps/README.md]: I'd prefer to stick to dashes only in class names.

**Comment by jpc-ae** [comment]
I've updated my branch with several changes and the documentation regarding use. I updated the index.html file as well and prettified it as best I could.

Regarding class names, I was following the conventions used throughout the project. I personally prefer to stick with dashes as well.

**Comment by jpc-ae** [comment]
I'm actually using a gulpfule to generate my files. If you want, I can either commit it or leave a comment. I wasn't sure how you generate your files since there are no scrips in the project folder.

**Comment by shyndman** [comment]
Could we get this modified to include only the docs? 

And thank you. I really appreciate you bringing symbol sprites to my attention.

**Comment by jpc-ae** [comment]
Sure. Just have to rebase and clean some things up.

**Pull request closed without merging**